docroot/modules/custom/foia_request/src/Form/FoiaRequestSettingsForm.php

```diff
@@ -8,7 +8,7 @@
 /**
  * Class FoiaRequestSettingsForm.
  *
- * @ingroup foia_request
+ * Form renders the FOIA request.
```

Comment:
Same comment here - do not remove `@` lines. Similar what I said above, the better fix would be:

```
/**
 * Render the FOIA request settings form.
 *
 * @ingroup foia_request
 */
```